script/c100100008.lua

```diff
@@ -1,100 +1,114 @@
 --Ｓｐ－ミラクルシンクロフュージョン
 function c100100008.initial_effect(c)
-	--Activate
-	local e1=Effect.CreateEffect(c)
-	e1:SetCategory(CATEGORY_SPECIAL_SUMMON+CATEGORY_FUSION_SUMMON)
-	e1:SetType(EFFECT_TYPE_ACTIVATE)
-	e1:SetCode(EVENT_FREE_CHAIN)
-	e1:SetCost(c100100008.cost)
-	e1:SetTarget(c100100008.target)
-	e1:SetOperation(c100100008.activate)
-	c:RegisterEffect(e1)
-	--draw
-	local e2=Effect.CreateEffect(c)
-	e2:SetDescription(aux.Stringid(100100008,1))
-	e2:SetCategory(CATEGORY_DRAW)
-	e2:SetProperty(EFFECT_FLAG_DAMAGE_STEP+EFFECT_FLAG_PLAYER_TARGET)
-	e2:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_TRIGGER_F)
-	e2:SetCode(EVENT_TO_GRAVE)
-	e2:SetCondition(c100100008.drcon)
-	e2:SetTarget(c100100008.drtg)
-	e2:SetOperation(c100100008.drop)
-	c:RegisterEffect(e2)
+    --Activate
```

Comment:
What kind of editor do you use? Please use one with proper formatting, Use a tab on each line instead of 4 space characters.